tests/tests_caching_logic.py

```diff
@@ -1,13 +1,18 @@
-from multiprocessing import Process
 import os
```

Comment:
Have you added this test to Github Action?

Reply:
Yes. The test is modified based on an original test and I verified it is executed.